calendar-bundle/src/Resources/contao/modules/ModuleEventReader.php

```diff
@@ -129,7 +129,15 @@ protected function compile()
 			case 'external':
 				if ($objEvent->url)
 				{
-					throw new RedirectResponseException($objEvent->url, 301);
+					$url = System::getContainer()->get('contao.insert_tag.parser')->replaceInline($objEvent->url);
+
+					// Make the URL absolute
+					if (!preg_match('@^https?://@i', $url))
+					{
+						$url = Environment::get('base') . ltrim($url, '/');
+					}
```

Comment:
```suggestion
					$url = UrlUtil::makeAbsolute($url, Environment::get('base'));
```

Reply:
In the `PageRedirect` class, the last argument is `Environment::get('base') . '/'`. Your suggestion does not contain a trailing slash though. Which version is correct?

Reply:
`PageRedirect` uses `Environment::get('path') . '/'` (without the scheme and domain part)
`Environment::get('base')` arleady has a trailing slash.
So both are correct, one with domain (absolute URL) the other one without (absolute path)

Reply:
Changed in 7ffccdf143c54a59fab808dedd690975b9baff30.